Apps/Sandcastle/gallery/CZML Different Coordinate Maps.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="CZML Point">
+    <meta name="cesium-sandcastle-labels" content="CZML">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+    <script type="text/javascript">
+        require.config({
+            baseUrl : '../../../Source',
+            waitSeconds : 60
+        });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+    function startup(Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+        var czml = [
+            {
+                "id" : "document",
+                "name" : "CZML Different Coordinate Maps",
+                "version" : "1.0"
+            },
+            {
+                "id" : "point 1",
+                name: "point1",
+                "position" : {
+                    "cartographicDegrees" : [-111.0, 40.0, 150000.0]
+                },
+                "point": {
+                    "color": {
+                        rgba: [100,0, 200, 255]
+                    },
+                    "outlineColor": {
+                        rgba: [200,0, 200, 255]
+                    },
+                    "pixelSize": {
+                        "number": 10
+                    },
+                    "show": true
+                }
+            }, {
+                "id" : "point 2",
+                name: "point2",
+                "position" : {
+                    "cartesian" : [
+                        1216469.9357990976, -4736121.71856379, 4081386.8856866374
+                    ]                },
+                "point": {
+                    "color": {
+                        rgba: [0,100, 200, 255]
```

Comment:
Also use consistent spacing: `[0, 100, 200, 255]` instead of `[0,100, 200, 255]`.